docs/preview/yamlgettingstarted-definition.md

```diff
@@ -2,9 +2,8 @@
 
 ## When using a Git repo in your VSTS team project
 
-Push a YAML file `.vsts-ci.yml` to the default branch (typically master) in the root
-of your repository. A definition `<REPO_NAME>/<REPO_NAME> CI` will be created with a
-continuous integration trigger for the default branch.
+Push a YAML file `.vsts-ci.yml` to the root directory of your repository. A definition
+`<REPO_NAME>/<REPO_NAME> CI` will be created, and a CI build will be triggered.
 
 Note, the definition will only be created if whoever pushes the branch update has
```

Comment:
What happens if someone without permissions were to push?  Is there a process by which authorization could be given after the push?

Reply:
Yes, if someone with permission pushes an update to the file. I'll clarify in master.

Reply:
Yes there is a way to authorize after the push.  We need to figure out a way to make this better with GitHub as we do not have the VSTS user context.

Reply:
@chrisrpatterson Can you use the github user info in that case?

Reply:
Without some other mapping stored in VSTS we don't know that a given GitHub user actually has permissions in VSTS.